fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadInfo.java

```diff
@@ -61,6 +61,10 @@ public class StreamLoadInfo {
     private String mergeConditionStr;
     private ColumnSeparator columnSeparator;
     private RowDelimiter rowDelimiter;
+    private long skipHeader;
```

Comment:
1. Why not encapsulate these format properties in a class like in Broker Load?
2. Why not abstract a class FileFormat, and implement CsvFileFormat, maybe later for JsonFileFormat, ParquetFileFormat, ...